r124/inref/ydb312gtm8182a.m

```diff
@@ -0,0 +1,42 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+gtm8182a
+	DO getPaths
+
+	WRITE "    Update the INST1 DB",!
+	SET ^today="7/18/2018"
+
+	WRITE "    Read from INST3 DB",!
+	WRITE "        Django: ",^|INST3gbldir|Django,!
+
+	;call jnlprocnum.csh to return the number or processes attached
+	;to our JNL pools
+	ZSYSTEM "$gtm_tst/r124/u_inref/jnlprocnum.csh"
+
+	quit
+
+updateINST3
+	DO getPaths
+	SET $ZGBLDIR=INST3gbldir
+	SET ^Django="Unchained"
+
+	quit
+
+getPaths
+	SET INST1path=$ZTRNLNM("path_INST1","","","","","VALUE")
```

Comment:
$ZTRNLNM("path_INST1") is enough. No need to pass the additional parameters. Please fix this in all subtests that have similar usages.